src/main/java/com/trzewik/jdbc/db/Db.java

```diff
@@ -0,0 +1,42 @@
+package com.trzewik.jdbc.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+class Db implements AutoCloseable {
+    private Connection connection;
+
+    Db(DbProperties properties) throws SQLException {
+        connection = DriverManager.getConnection(
+            properties.getUrl(),
+            properties.getUsername(),
+            properties.getPassword());
+    }
+
+    void startTransaction() throws SQLException {
+        connection.setAutoCommit(false);
```

Comment:
to jest start transakcji? hmmm

Reply:
tak jakby? :D 
https://docs.oracle.com/javase/tutorial/jdbc/basics/transactions.html